community_images/node-exporter/ironbank/k8s_coverage.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -x
+set -e
+# shellcheck disable=SC1091
+SCRIPTPATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
+
+# shellcheck disable=SC1091
+. "${SCRIPTPATH}"/../../common/scripts/bash_helper.sh
+
+JSON_PARAMS="$1"
+
+JSON=$(cat "$JSON_PARAMS")
+
+echo "Json params for k8s coverage = $JSON"
+
+NAMESPACE=$(jq -r '.namespace_name' < "$JSON_PARAMS")
+RELEASE_NAME=$(jq -r '.release_name' < "$JSON_PARAMS")
+echo "${RELEASE_NAME}"
+CONTAINER_NAME="${RELEASE_NAME}"
+kubectl logs -n "${NAMESPACE}" "${CONTAINER_NAME}"
```

Comment:
@Rajat0002 you can remove `kubectl logs -n ...` as its already present in orchestrator.

Reply:
Done